mapclassify/value_by_alpha.py

```diff
@@ -0,0 +1,533 @@
+    ...                rgb=dict(classifier='quantiles', k=3),
+    ...                alpha=dict(classifier='quantiles'))  # doctest: +SKIP
+    >>> plt.show()  # doctest: +SKIP
+    >>> plt.close()
+
+    Add a legend and use divergent alpha values
+
+    >>> fig = plt.figure(figsize=(15,10))
+    >>> ax = fig.add_subplot(111)
+    >>> vba_choropleth('HOVAL', 'CRIME', gdf, divergent=True,
+    ...                alpha=dict(classifier='quantiles', k=5),
+    ...                rgb=dict(classifier='quantiles', k=5),
+    ...                legend=True, ax=ax,
+    ...                legend_kwargs={"alpha_label": "CRIME", "rgb_label": "HOVAL"})  # doctest: +SKIP
+    >>> plt.show()  # doctest: +SKIP
+    >>> plt.close()
+
+    """  # noqa: E501
+
+    if not MPL_AVAILABLE:
+        raise MPL_NOT_AVAILABLE
+
+    x = gdf[x_var].to_numpy() if isinstance(x_var, str) else x_var
+    y = gdf[y_var].to_numpy() if isinstance(y_var, str) else y_var
+
+    if ax is None:
+        fig = plt.figure()
+        ax = fig.add_subplot(111)
+    else:
+        fig = ax.get_figure()
+
+    if rgb is not None:
+        rgb.setdefault("k", 5)
+        rgb.setdefault("hinge", 1.5)
+        rgb.setdefault("multiples", [-2, -1, 1, 2])
+        rgb.setdefault("mindiff", 0)
+        rgb.setdefault("initial", 100)
+        rgb.setdefault("bins", [20, max(x)])
+        classifier = rgb["classifier"]
+        k = rgb["k"]
+        hinge = rgb["hinge"]
+        multiples = rgb["multiples"]
+        mindiff = rgb["mindiff"]
+        initial = rgb["initial"]
+        bins = rgb["bins"]
+        rgb_bins = classify(
+            x,
+            classifier,
+            k=k,
+            hinge=hinge,
+            multiples=multiples,
+            mindiff=mindiff,
+            initial=initial,
+            bins=bins,
+        )
+        x = rgb_bins.yb
+
+    if alpha is not None:
+        alpha.setdefault("k", 5)
+        alpha.setdefault("hinge", 1.5)
+        alpha.setdefault("multiples", [-2, -1, 1, 2])
+        alpha.setdefault("mindiff", 0)
+        alpha.setdefault("initial", 100)
+        alpha.setdefault("bins", [20, max(y)])
+        classifier = alpha["classifier"]
+        k = alpha["k"]
+        hinge = alpha["hinge"]
+        multiples = alpha["multiples"]
+        mindiff = alpha["mindiff"]
+        initial = alpha["initial"]
+        bins = alpha["bins"]
+        # TODO: use the pct keyword here
+        alpha_bins = classify(
+            y,
+            classifier,
+            k=k,
+            hinge=hinge,
+            multiples=multiples,
+            mindiff=mindiff,
+            initial=initial,
+            bins=bins,
+        )
+        y = alpha_bins.yb
+
+    rgba, vba_cmap = _value_by_alpha_cmap(
+        x,
+        y,
+        cmap=cmap,
+        divergent=divergent,
+        revert_alpha=revert_alpha,
+        min_alpha=min_alpha,
+    )
+    gdf.plot(color=rgba, ax=ax)
+    ax.set_axis_off()
+    ax.set_aspect("equal")
```

Comment:
```suggestion
```

let geopandas deal with the aspect. This will break aspect set for geographic coords.